atd-vzd/triggers/is_crash_level_5.sql

```diff
@@ -0,0 +1,9 @@
+-- This function determines whether crash occurred on a level 5 road (large highway)
+-- and is thus outside of COA jurisdiction.
+CREATE OR REPLACE FUNCTION is_crash_level_5(crash_id integer, rpt_road_part_id integer, rpt_hwy_num text) RETURNS boolean AS $$
+        BEGIN
+        RETURN (
+            SELECT(rpt_road_part_id != 2 AND UPPER(LTRIM(rpt_hwy_num)) IN ('35', '183','183A','1','290','71','360','620','45','130'))
```

Comment:
based on logic from https://github.com/cityofaustin/atd-data-tech/issues/11368 where xavier defines what a level 5 crash is

Reply:
something i just thought of tho - what if the rpt_road_part_id or rpt_hwy_num is wrong and since those arent values the VZ team can fix/update, then the crash is just wrongfully identified as level 5 and isnt given a location?

Reply:
I think that it is. When we talked with Xavier in the past, he imagined being able to edit those columns, but, when Frank and I synced on it afterward, we thought about another editable boolean column like `is_level_5` that they could edit to push a crash into the accurate category. Hoping that we can cover that in the meeting on Monday morning to get a feel for it!